compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -320,19 +320,18 @@ impl<'tcx> CodegenUnit<'tcx> {
         base_n::encode(hash, base_n::CASE_INSENSITIVE)
     }
 
-    pub fn create_size_estimate(&mut self, tcx: TyCtxt<'tcx>) {
+    pub fn compute_size_estimate(&mut self, tcx: TyCtxt<'tcx>) {
         // Estimate the size of a codegen unit as (approximately) the number of MIR
         // statements it corresponds to.
-        self.size_estimate = Some(self.items.keys().map(|mi| mi.size_estimate(tcx)).sum());
+        self.size_estimate = self.items.keys().map(|mi| mi.size_estimate(tcx)).sum();
     }
 
     #[inline]
-    /// Should only be called if [`create_size_estimate`] has previously been called.
+    /// Should only be called if [`compute_size_estimate`] has previously been called.
     ///
-    /// [`create_size_estimate`]: Self::create_size_estimate
+    /// [`compute_size_estimate`]: Self::compute_size_estimate
```

Comment:
Nit: you can still detect the case where `compute_size_estimate` was not called before this function was by asserting that if `items.len() > 0`, then `size != 0`. The minimum possible size of a `MonoItem` is `1` so it is impossible for `size_estimate` to actually be `0` if there are any items.

Reply:
Good suggestion. I will add `assert!(self.items.is_empty() || self.size_estimate != 0);`.